src/Engine.cpp

```diff
@@ -1482,7 +1489,16 @@ void Engine::set_alias(
 
     if (last_entities_clicked_.dds.id == entity_id)
     {
-        info_model_->update_selected_entity(backend::backend_id_to_models_id(entity_id), utils::to_QString(new_alias));
+        EntityInfo entity_info = backend_connection_.get_info(entity_id);
+        std::string app_id = "UNKNOWN_APP";
+        if (entity_info.contains("app_id"))
+        {
+            app_id = entity_info["app_id"];
+        }
+        info_model_->update_selected_entity(
+                utils::to_QString(entity_info["kind"]),
```

Comment:
If kind + alias is too large, the total displayed text isn't completely shown.
If workaround isn't direct, solve it in a different PR.